src/replica/replica.cpp

```diff
@@ -313,6 +314,8 @@ void replica::execute_mutation(mutation_ptr &mu)
     }
 
     // update table level latency perf-counters for primary partition
+    mu->tracer->add_point("complete(write2rocksdb)");
+    mu->tracer->dump_trace_points(FLAGS_abnormal_write_trace_latency_threshold);
```

Comment:
Though you have dumped this tracer here, but points in tracer are still valid, how to prevent it from dumping duplicatly?
So how about dump trace points in tracer's deconstructor automatically?

Reply:
`aio_task` and `mutation` both have `tracer`,  `dump trace points in tracer's deconstructor automatically` will be duplicatly

Reply:
You can mute sub-tracer's dump log in some way when you explicitly add it to another trace.
And how about the first question?

Reply:
I will accept the suggestion `dump_trace_points()` in tracer's deconstructor automatically, it do will prevent it from dumping duplicatly